script/profile_db_generator.rb

```diff
@@ -64,20 +68,9 @@ def create_admin(seq)
   exit
 end
 
-def ensure_perf_test_topic_has_right_title!
-  title = "I am a topic used for perf tests"
-  # in case we have an old run and picked the wrong topic
-  Topic.where(title: title).update_all(title: "Test topic #{SecureRandom.hex}")
-  t = Topic.where(archetype: :regular, posts_count: 30).order(id: :desc).first
-  t.title = title
-  t.save!
-end
```

Comment:
Redundant now that we will always create a topic with 100 posts which will be used as the topic to benchmark against.